pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/source/PulsarSource.java

```diff
@@ -80,17 +80,17 @@ public void open(Map<String, Object> config, SourceContext sourceContext) throws
                     .messageListener(this);
 
             if (conf.isRegexPattern) {
-                cb.topicsPattern(topic);
+                cb = cb.topicsPattern(topic);
```

Comment:
The assignment is not needed, the builder would update in-place

Reply:
This is needed for test, since otherwise the mock would return null